packages/notebook-extension/schema/tracker.json

```
@@ -699,6 +699,12 @@
       "type": "boolean",
       "default": false
     },
+    "showInputPlaceholder": {
+      "title": "Show input placeholder",
+      "description": "Show placeholder text for standard input fields. (Needs reload)",
```

@JasonWeill what do you think would be the best wording here?

```suggestion
      "description": "Show placeholder text for standard input fields (requires reloading JupyterLab to take effect)",
```

---

I think that (needs reload) or (requires reload) is both informative and concise. The longer phrase would be better for screen reader users, though.

---

@krassowski Should I change the description?